### PR TITLE
DW-5386 added passed param to prod and preprod ci pipelines and remov…

### PR DESCRIPTION
**Pull request by sandeepg-github**
…ed count on tf resources

### CODE REVIEW

File: data-egress.tf [hunk 1]
```diff
@@ -114,7 +110,7 @@ data "aws_iam_policy_document" "published_non_sensitive_bucket_s3" {
 }
 
 resource "aws_acm_certificate" "data_egress_server" {
-  count                     = local.is_mgmt_env[local.environment] ? 0 : 1
+  count      = local.is_mgmt_env[local.environment] ? 0 : 1
```
Comment (dan-hill2802) [inline data-egress.tf:113]: this doesn't seem right, the previous indent aligned the equals symbols. Could your run `terraform fmt` to confirm?

File: locals.tf [hunk 2]
```diff
@@ -6,56 +6,44 @@ locals {
     integration    = false
     preprod        = false
     production     = false
-    management     = true
```
Comment (dan-hill2802) [inline locals.tf:9]: you should be able to remove this `is_mgmt_env ` block now as it looks like it might be unused

File: data-egress.tf (hunk 1 above)
Comment (sandeepg-github) [reply data-egress.tf]: done

File: locals.tf (hunk 2 above)
Comment (sandeepg-github) [reply locals.tf:9]: removed

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**